.github/ISSUE_TEMPLATE/breaking_change.yml

```diff
@@ -0,0 +1,70 @@
+# This issue template is for use in opening issues that document breaking changes. This template should be used to create an issue by Microsoft product team members who are documenting a breaking change.
+# 
+# Bracketed text is a placeholder; replace the text with the requested information and remove the brackets before submitting the issue. Also, remove this comment before submitting the issue.
+
+name: "Identity Web breaking change"
```

Comment:
not necessarily breaking changes. this is more for any code that needs to go in between the preview and the release...not such an issue in Id web and IdentityModel as we don't ship those in preview much, but would like to have the same form.

We need a different title

Reply:
Resolved comments and reworked in the wilson PR, should be brought up to date now!